package.json

```diff
@@ -50,6 +50,7 @@
   "homepage": "https://github.com/dougmoscrop/serverless-http",
   "license": "MIT",
   "optionalDependencies": {
+    "@azure/functions": "^1.0.3",
```

Comment:
Can this (as well as the other @types) be a devDependency? I don't use TypeScript so I am not sure when this is needed, but ideally we don't try to install this for everyone?

Reply:
Yes, they can. The @types are only needed when using typescript, otherwise, not needed at all.

Reply:
If you could move them I'm good to merge, otherwise I can merge and do it instead..

Reply:
Please could this be resolved? I'm quite keen to see this functionality available for Azure! If you'd like an assist I'm happy to help.